iot/iot-device-registry-jdbc/src/main/java/io/enmasse/iot/registry/jdbc/config/DeviceConnectionServiceConfiguration.java

```diff
@@ -18,7 +19,6 @@
 import io.vertx.core.Vertx;
 
 @Configuration
-@Profile(PROFILE_DEVICE_CONNECTION)
```

Comment:
I think dropping the profile here and the component annotation on the service, will break the deployment, as the operator can no longer properly fire up specific services.